Sources/Moya/Plugins/AccessTokenPlugin.swift

```diff
@@ -6,47 +6,49 @@ import Result
 /// A protocol for controlling the behavior of `AccessTokenPlugin`.
 public protocol AccessTokenAuthorizable {
 
-    /// Represents the authorization header to use for requests.
+    /// Represents the authorization header or url parameter to use for requests.
     var authorizationType: AuthorizationType { get }
 }
 
 // MARK: - AuthorizationType
 
-/// An enum representing the header to use with an `AccessTokenPlugin`
+/// An enum representing the header and url parameter to use with an `AccessTokenPlugin`
```

Comment:
Very minor detail but "and url parameter" -> "or url parameter"

Reply:
Hello @SD10 , thank you for help. I will make some fixes at evening 👍